operator/cmd/operator/server.go

```diff
@@ -134,6 +140,15 @@ func run() {
 		log.Fatalf("Could not create a controller manager: %v", err)
 	}
 
+	log.Info("Creating operator metrics exporter")
+	exporter, err := ocprom.NewExporter(ocprom.Options{
+		Registry: ctrlmetrics.Registry.(*prometheus.Registry),
+	})
+	if err != nil {
+		log.Fatalf("Error while building exporter")
```

Comment:
is this truly a `Fatal` error?  Would this be better as a `Warn`?  And can we include the error that generated in the formatted output via `%v` or similar ?

Reply:
Fixed this.